src/Nethermind/Nethermind.JsonRpc/Modules/Parity/ParityTransaction.cs

```diff
@@ -60,8 +60,8 @@ public ParityTransaction(Transaction transaction, byte[] raw, PublicKey publicKe
             Input = transaction.Data.AsArray();
             PublicKey = publicKey;
             ChainId = transaction.Signature.ChainId;
-            R = transaction.Signature.R;
-            S = transaction.Signature.S;
+            R = transaction.Signature.R.ToArray();
+            S = transaction.Signature.S.ToArray();
```

Comment:
We could back it up by signature object and make R and S spans here?

Reply:
Switched to Memory